lib/ex_aws/sqs/parsers.ex

```diff
@@ -119,11 +119,17 @@ if Code.ensure_loaded?(SweetXml) do
                           ]
                         ])
 
-      fixed_attributes = parsed_body
-      |> fix_attributes([:message, :attributes], &attribute_list_to_map/1)
-      |> fix_attributes([:message, :message_attributes], &message_attributes_to_map/1)
+      new_messages = parsed_body
+      |> Map.get(:messages)
```

Comment:
Since `:messages` is marked as optional in theory `Map.get(parsed_body, :messages)` could return `nil`. We either want it non optional or with a `|> Map.get(:messages, [])`